SampleCode/README.md

```diff
@@ -21,10 +17,17 @@ Refer to [this repo](https://github.com/Azure/azure-sdk-for-java/tree/azure-ai-f
 # Samples for JavaScript SDK
```

Comment:
same for the JAVA and JavaScript versions

Reply:
Updated.